### DISCUSSION TITLE
Toggle The Player

### DISCUSSION THREAD

**Discussion by mcquaim** [alandtse/alexa_media_player#1387] [Q&A]
Hi guys,

Is it possible to have the player to toggle from on to off?

So, I have an automation where if I press a button my office light come on and the Echo Dot starts to play my usual radio station from Tunein.

The light is a toggle so if I press it initially then it goes on, then off with the next press etc.

Can something similar be achieved with the Alexa Media Player? First press then play the station, second, then stop if it's playing?

Cheers,
Mac

**Comment by alandtse** [comment]
I'm not sure what you're toggling. Your light? Then yes if you know how to script it in HA.
